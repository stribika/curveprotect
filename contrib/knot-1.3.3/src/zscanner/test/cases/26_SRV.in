$ORIGIN	.
$TTL	1

; OK
@	SRV	0	0	0	@			; The simplest variant
_ldap._tcp.test.tld.	SRV	0	0	0	@ 	; Underscores in owner
@	SRV	65535	65535	65535	@			; Maximal numbers
@	SRV	0	0	0	\0320\ \\\"\.\@\*.tld.	; Dname with specials
@	TYPE33	\# 7 000000000000 00				; TYPE + Hexadecimal rdata
@	TYPE33	0	0	0	@			; TYPE
@	srv	0	0	0	@			; Type in lower-case

; KO
@	SRV
@	SRV						; Empty rdata
@	SRV	\# 0					; Hex empty rdata
@	SRV	\#					; Missing hex length
@	SRV	1h	0	0	@		; Bad priority
@	SRV	0	1h	0	@		; Bad weight
@	SRV	0	0	1h	@		; Bad port
@	SRV	0	0	0	%		; Bad target
@	SRV	65536	0	0	@		; Priority overflow
@	SRV	0	65536	0	@		; Weight overflow
@	SRV	0	0	65536	@		; Port overflow
@	SRV	0	0	0	@ x		; Unexpected item
@	SRV	0	0	0			; Missing item
@	SRV	\# 8 000000000000 00 00			; Too long rdata
@	SRV	\# 8 000000000000 00			; Bad rdata length
