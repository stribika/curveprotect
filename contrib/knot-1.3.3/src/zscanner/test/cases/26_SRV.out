OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0021
RDATA=000000000000 00
------
OWNER=055F6C646170045F746370047465737403746C6400
CLASS=0001
RRTTL=00000001
RTYPE=0021
RDATA=000000000000 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0021
RDATA=FFFFFFFFFFFF 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0021
RDATA=000000000000 082030205C222E402A03746C6400
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0021
RDATA=000000000000 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0021
RDATA=000000000000 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0021
RDATA=000000000000 00
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_EBAD_REST
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
